apps/builder/app/dashboard/loading-state/loading.tsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from "react";
+import { Flex, css } from "@webstudio-is/design-system";
+import { SvgLoading } from "~/dashboard/loading-state/svg-loading";
+import { WebmLoading } from "~/dashboard/loading-state/webm-loading";
+
+const containerStyle = {
+  position: "absolute",
+  left: "0",
+  right: "0",
+  top: "0",
+  bottom: "0",
+};
+
+const lightContainerStyle = css(containerStyle, {
+  backgroundColor: "rgb(255 255 255 / 50%);",
+});
+const darkContainerStyle = css(containerStyle, {
+  backgroundColor: "rgb(0 0 0 / 80%);",
+});
+
+export const Loading = ({
+  type = "webm",
+  isDark = false,
+  delay = 0,
+}: {
+  type?: "webm" | "svg";
+  isDark?: boolean;
```

Comment:
we will only need one version before merging

Reply:
Updated [here](https://github.com/webstudio-is/webstudio-builder/pull/1579/commits/0146d766c5cf139211640b7fb505b505c6a91d05). We only use the light overlay  and SVG now.